### ISSUE TITLE
TypeError: Point.row must be a number

### ISSUE THREAD

**Issue by hendrikbl** [susisu/atom-markdown-table-editor#18]
1. Enable *Use Tree Sitter Parser* in core settings
2. Activate markdown-table-editor plugin

**Atom**: 1.25.0-beta1 x64
**Electron**: 1.7.11
**OS**: Windows 7 Enterprise
**Thrown From**: [markdown-table-editor](https://github.com/susisu/atom-markdown-table-editor) package 1.0.2

### Stack Trace

Failed to activate the markdown-table-editor package

```
At Point.row must be a number

TypeError: Point.row must be a number
    at TreeSitterLanguageMode.scopeDescriptorForPosition (~/AppData/Local/atom/app-1.25.0-beta1/resources/app/src/tree-sitter-language-mode.js:318:45)
    at TextEditor.scopeDescriptorForBufferPosition (~/AppData/Local/atom/app-1.25.0-beta1/resources/app/src/text-editor.js:3663:48)
    at TextEditorInterface.acceptsTableEdit (/packages/markdown-table-editor/lib/text-editor-interface.js:35:32)
    at TableEditor.cursorIsInTable (/packages/markdown-table-editor/node_modules/@susisu/mte-kernel/dist/mte-kernel.js:1973:29)
    at EditorController.isActive (/packages/markdown-table-editor/lib/editor-controller.js:58:66)
    at /packages/markdown-table-editor/lib/editor-controller.js:15:33)
    at Controller.addEditor (/packages/markdown-table-editor/lib/controller.js:113:26)
    at Controller.subscriptions.add.atom.workspace.observeTextEditors.editor (/packages/markdown-table-editor/lib/controller.js:21:12)
    at Workspace.observeTextEditors (~/AppData/Local/atom/app-1.25.0-beta1/resources/app/src/workspace.js:637:59)
    at /packages/markdown-table-editor/lib/controller.js:20:43)
    at MarkdownTableEditor.activate (/packages/markdown-table-editor/lib/markdown-table-editor.js:11:23)
    at Package.activateNow (~/AppData/Local/atom/app-1.25.0-beta1/resources/app/src/package.js:230:33)
    at measure (~/AppData/Local/atom/app-1.25.0-beta1/resources/app/src/package.js:206:33)
    at Package.measure (~/AppData/Local/atom/app-1.25.0-beta1/resources/app/src/package.js:88:25)
    at activationPromise.Promise (~/AppData/Local/atom/app-1.25.0-beta1/resources/app/src/package.js:200:20)
    at Promise (<anonymous>)
    at Package.activate (~/AppData/Local/atom/app-1.25.0-beta1/resources/app/src/package.js:198:38)
    at PackageManager.activatePackage (~/AppData/Local/atom/app-1.25.0-beta1/resources/app/src/package-manager.js:695:42)
    at packagesToEnable.forEach.name (~/AppData/Local/atom/app-1.25.0-beta1/resources/app/src/package-manager.js:440:51)
    at Array.forEach (native)
    at disabledPackagesSubscription.config.onDidChange (~/AppData/Local/atom/app-1.25.0-beta1/resources/app/src/package-manager.js:440:30)
    at emitter.on (~/AppData/Local/atom/app-1.25.0-beta1/resources/app/src/config.js:1013:22)
    at Function.module.exports.Emitter.simpleDispatch (~/AppData/Local/atom/app-1.25.0-beta1/resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (~/AppData/Local/atom/app-1.25.0-beta1/resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at Config.emitChangeEvent (~/AppData/Local/atom/app-1.25.0-beta1/resources/app/src/config.js:1165:62)
    at Config.setRawValue (~/AppData/Local/atom/app-1.25.0-beta1/resources/app/src/config.js:998:23)
    at Config.set (~/AppData/Local/atom/app-1.25.0-beta1/resources/app/src/config.js:711:18)
    at Config.removeAtKeyPath (~/AppData/Local/atom/app-1.25.0-beta1/resources/app/src/config.js:891:16)
    at Package.enable (~/AppData/Local/atom/app-1.25.0-beta1/resources/app/src/package.js:75:30)
    at PackageManager.enablePackage (~/AppData/Local/atom/app-1.25.0-beta1/resources/app/src/package-manager.js:246:18)
    at HTMLButtonElement.enablementButtonClickHandler (~/AppData/Local/atom/app-1.25.0-beta1/resources/app/node_modules/settings-view/lib/package-card.js:362:31)
```

**Comment by susisu** [comment]
Thank you for reporting!
I found there is a breaking change (probably not intended) in Atom API, and will report it later.

Please disable `Use Tree Sitter Parser` for a while...

---

related codes:
* https://github.com/atom/atom/blob/v1.25.0-beta1/src/text-editor.js#L3646-L3659
* https://github.com/atom/atom/blob/v1.25.0-beta1/src/tree-sitter-language-mode.js#L300
* https://github.com/tree-sitter/node-tree-sitter/blob/b2846701e980572ef287e0effe54d4a91e3c1075/src/ast_node.cc#L281
* https://github.com/tree-sitter/node-tree-sitter/blob/6aeac7a04e817006fbc59a4b4b63f7cbc150a1fa/src/conversions.cc#L46-L50

**Comment by susisu** [comment]
It is fixed in v1.25.0-beta2 so I close.

**Comment by hendrikbl** [comment]
Nice, thanks for looking into it :)
